internal/control-panel/registry.lua

```diff
@@ -125,6 +125,8 @@ COMMANDS_BY_IDX = {
     {command='partial-items', help_command='tweak', group='gameplay', mode='tweak', default=true,
         desc='Displays percentages on partially-consumed items like hospital cloth.'},
     {command='pop-control', group='gameplay', mode='enable'},
+    {command='realistic-melting', help_command='tweak', group='gameplay', mode='tweak', default=false,
```

Comment:
you can leave off the `default=false` part -- we only add it if it's set to `true`

Reply:
ok, removed